### PR TITLE
Optimize the code for the callback module

### PR DESCRIPTION
**Pull request by kerwin612**
##### SUMMARY

Optimize the code for the callback module
Extract the consistent code to the parent class, the child class share the parent class method

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- callback

### CODE REVIEW

File: lib/ansible/plugins/callback/__init__.py [hunk 1]
```diff
@@ -98,6 +98,9 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
         # load from config
         self._plugin_options = C.config.get_plugin_options(get_plugin_class(self), self._load_name, keys=task_keys, variables=var_options, direct=direct)
 
+    def _run_is_verbose(self, result):
+        return ((self._display.verbosity > 0 or '_ansible_verbose_always' in result._result) and '_ansible_verbose_override' not in result._result)
```
Comment (bcoca) [inline lib/ansible/plugins/callback/__init__.py]: I know this is preexisting issue, but the condition should be checking the boolean value, not just the existence

**Review by bcoca — CHANGES_REQUESTED** [review]

File: lib/ansible/plugins/callback/__init__.py (hunk 1 above)
Comment (kerwin612) [reply lib/ansible/plugins/callback/__init__.py]: get it

**Comment by ileler** [comment]
ready_for_review

**Comment by ileler** [comment]
ready_for_review

**Comment by ileler** [comment]
Excuse me, what else do I need to do?@bcoca

**Review by bcoca — APPROVED** [review]

**Comment by bcoca** [comment]
rebuild_merge

**Pull request merged**